specs/opcode/54SLOAD_55SSTORE.md

```diff
@@ -0,0 +1,89 @@
+# SLOAD & SSTORE op code
+
+Disclaimer: this version of the sload/sstore sepc only corresponds to [EIP-2929](https://eips.ethereum.org/EIPS/eip-2929).
+
```

Comment:
but does not implement EIP 2930 yet? (if yes, add this into the spec)

Reply:
none of 2028, 2930, 1559 is  implemented yet. should I add them all?

Reply:
in fact. 2930 breaks (improves on) 2029. if 2930 then it can't exactly be 2929

Reply:
fixed in b1db9761011f536acd25913b29c26425adf423e3